### PR TITLE
Minimal threading docs

### PR DESCRIPTION
**Pull request by arnetheduck**

### CODE REVIEW

File: docs/src/threads.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+parallel with the async work, or, in cases where a single event loop gets
+overloaded, to manage multiple event loops in parallel.
+
+For interaction between threads, the `ThreadSignalPtr` type (found in the
+(`chronos/threadsync`)(https://github.com/status-im/nim-chronos/blob/master/chronos/threadsync.nim)
+module) is used - both to wait for notifications coming from other threads and
+to notify other threads of progress from within an async procedure.
+
+```nim
+{{#include ../examples/twogets.nim}}
```
Comment (kdeme) [inline docs/src/threads.md:17]: wrong example?

File: docs/src/examples.md [hunk 2]
```diff
@@ -8,6 +8,10 @@ Examples are available in the [`docs/examples/`](https://github.com/status-im/ni
 * [timeoutsimple](https://github.com/status-im/nim-chronos/tree/master/docs/examples/timeoutsimple.nim) - Simple timeouts
 * [timeoutcomposed](https://github.com/status-im/nim-chronos/tree/master/docs/examples/examples/timeoutcomposed.nim) - Shared timeout of multiple tasks
 
+## Threads
+
+* [signalling](https://github.com/status-im/nim-chronos/tree/master/docs/examples/cancellation.nim) - Cross-thread signalling
```
Comment (kdeme) [inline docs/src/examples.md:13]: wrong link to example?

File: docs/src/threads.md (hunk 1 above)
Comment (arnetheduck) [reply docs/src/threads.md]: thanks, caught me there ;)

Comment (arnetheduck) [reply docs/src/threads.md]: ec7a72d00cc88f058f449db60601d0390765e360

**Review by cheatfate — APPROVED** [review]

**Pull request merged**